### PR TITLE
refactor(www): Change the way we determine locales

### PR DESCRIPTION
**Pull request by tesseralis**
## Description

* change the `ENABLE_LOCALIZATIONS` variable to `LOCALES` and allow specifying a list of locales to compile
* add all currently existing translation efforts to config, and update the [translation guide](https://www.gatsbyjs.org/contributing/translation/) to use a list referencing it.

## Motivation

This gives us a single source of truth for current translation efforts that we can use in scripts, UIs, etc., while giving us a way to only enable building specific languages.

### CODE REVIEW

File: www/src/utils/i18n.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
-const langs = require("../../i18n.json")
+const allLangs = require("../../i18n.json")
 const defaultLang = "en"
 
-const langCodes = langs.map(lang => lang.code)
+// Only allow languages defined in the LOCALES env variable
+// This allows us to test only languages that are "complete" or test only
+// a single language
+const langCodes = process.env.LOCALES ? process.env.LOCALES.split(" ") : []
+const langs = allLangs.filter(lang => langCodes.includes(lang.code))
```
Comment (madalynrose) [inline www/src/utils/i18n.js]: should we add a warning if one of the `LOCALES` they've entered does not appear in the `langCodes` list or is invalid?

File: www/README.md [hunk 2]
```diff
@@ -42,13 +42,13 @@ ANALYTICS_SERVICE_ACCOUNT_KEY="PEM KEY VALUE"
 
 ### Enabling localizations
 
-Localizations are currently a work-in-progress and are thus disabled by default. They can be enabled by setting the `ENABLE_LOCALIZATIONS` env variable:
+Localizations are currently a work-in-progress and are thus disabled by default. They can be enabled by setting the `LOCALES` env variable to the locales you want to build:
```
Comment (LekoArts) [inline www/README.md]: I'd link to the json file here for reference 👍 
At least I don't know all language codes by hard 😆

Comment (tesseralis) [reply www/README.md]: what, you don't have the full list of ISO 639-1 language codes memorized? :P

File: www/src/utils/i18n.js (hunk 1 above)
Comment (tesseralis) [reply www/src/utils/i18n.js]: console: fart is not a valid locale

**Pull request merged**